windows/src/global/delphi/debug/Keyman.System.DebugLogCommon.pas

```diff
@@ -0,0 +1,36 @@
+unit Keyman.System.DebugLogCommon;
+
+interface
+
+const
+  // GUID that identifies the provider that you want
+  // to enable to your session.
+  DebugLogProviderGuid: TGUID = '{DA621615-E08B-4283-918E-D2502D3757AE}';
+
+function ShouldDebug: Boolean;
+
+implementation
+
+uses
+  System.Win.Registry,
+
+  RegistryKeys;
+
+var
+  FShouldDebug: Boolean;
+
+function ShouldDebug: Boolean;
+begin
+  Result := FShouldDebug;
```

Comment:
The intention in a future PR is to eliminate this flag and use ETW trace status instead. That would allow us to control flags for the tracing using xperf, e.g. https://blogs.msdn.microsoft.com/ntdebugging/2010/06/08/hidden-etw-stack-trace-feature-get-stacks-all-over-the-place/